bin/web-ui/main.ml

```diff
@@ -425,6 +425,11 @@ let onload _event =
             match code_formatted with
             | Ok code_formatted ->
                 code_input##.value := Js.string code_formatted
+                (* somehow, we have to replace above line with
+                let formatted = Js.string code_formatted in
+                (* editor is attached to window object *)
+                editor.doc.setValue(formatted)
+                *)
```

Comment:
I don't understand this comment. Is there something to do?

Reply:
@Khady thanks for looking into, I put out this just to gauge interest.
I don't have much experience using ocaml and I'm self learning. I saw your announcement in [discuss thread](https://discuss.ocaml.org/t/ann-ocamlformat-web-configurator/10103) and it's a really useful tool.

With that said, I haven't used `js_of_ocaml` so i dont know how to call `editor.doc.setValue` in the window object which is needed to update the editor with newly formatted string. That's why i left that comment.

Side note: I couldn't build the project so if there's an instruction how to build the project, I'll be happy to fix this.

Reply:
ok I see the problem now. But I don't know the immediate solution ^^' Will try to have a look

> Side note: I couldn't build the project so if there's an instruction how to build the project, I'll be happy to fix this.

That is a good remark, I should add a readme

Reply:
Ps: i updated but it fails if the input code has backtick (`), will look into it later